### DISCUSSION TITLE
Is it possible to use GraphQL in Landingpage?

### DISCUSSION THREAD

**Discussion by asemandeveloper** [OrchardCMS/OrchardCore#12905] [Q&A]
I want to use GraphQL in my landing page, for example this GraphQL statement

`query MyQuery {
  blogPost {
    displayText
    subtitle
  }
}`

is it possible?
please guide me

**Comment by hyzx86** [comment]
try make a get request via javascript

```
const result = await fetch("/api/graphql?query=query MyQuery { blogPost { displayText subtitle } }")
```
